packages/babel-helper-create-class-features-plugin/test/fixtures/plugin-proposal-class-properties/tdz-error-computed-prototypes/options.json

```diff
@@ -0,0 +1,10 @@
+{
+  "plugins": [
+    [
+      "proposal-class-properties",
+      {
+        "loose": true
```

Comment:
Can you also add a test in non-loose mode?

Reply:
Added them. PTAL.